lib/adapter.js

```diff
@@ -48,7 +51,8 @@ class CucumberAdapter {
     async run () {
         let reporterOptions = {
             capabilities: this.capabilities,
-            ignoreUndefinedDefinitions: !!this.cucumberOpts.ignoreUndefinedDefinitions
+            ignoreUndefinedDefinitions: !!this.cucumberOpts.ignoreUndefinedDefinitions,
```

Comment:
Let's do `Boolean(this.cucumberOpts.ignoreUndefinedDefinitions)` the same with the line below